Composer/packages/client/src/CreationFlow/CreateOptions/index.js

```diff
@@ -27,83 +47,120 @@ export function CreateOptions(props) {
     );
   }
 
-  function TemplateItem(props) {
-    const { checked, text, key, disabled } = props;
-    return (
-      <div key={key} css={templateItem(checked, disabled)}>
-        <div style={{ marginLeft: '5px' }}>{text}</div>
-      </div>
-    );
-  }
-
-  function onRenderField(props) {
-    const { checked } = props;
-    return (
-      <Fragment>
-        <SelectOption {...props} />
-        <ChoiceGroup
-          styles={choiceGroup}
-          options={templates.map(item => ({
-            ariaLabel: item.id,
-            'data-testid': item.id,
-            key: item.id,
-            text: item.name,
-            description: item.description,
-            onRenderField: TemplateItem,
-            disabled: !checked,
-          }))}
-          onChange={handleItemChange}
-          required={true}
-        />
-        <div css={placeholder}>
-          {checked && template ? template.description : formatMessage('Select a template to see its description.')}
-        </div>
-      </Fragment>
-    );
-  }
-
   const handleChange = (event, option) => {
     setOption(option.key);
-  };
-
-  const handleItemChange = (event, option) => {
-    setTemplate(option.key);
+    if (option.key === 'CreateFromTemplate') {
+      setDisabled(false);
+    } else {
+      setDisabled(true);
+    }
   };
 
   const handleJumpToNext = () => {
-    if (option === 'Create from template') {
+    if (option === 'CreateFromTemplate') {
       onNext(template);
     } else {
       onNext(emptyBotKey);
     }
   };
 
+  const tableColums = [
+    {
+      key: 'column1',
+      name: formatMessage('Name'),
+      fieldName: 'name',
+      minWidth: 150,
+      maxWidth: 200,
+      isResizable: !disabled,
+      data: 'string',
+      styles: rowTitle(disabled),
+      onRender: item => item.name,
+    },
+    {
+      key: 'column2',
+      name: formatMessage('Description'),
+      fieldName: 'dateModifiedValue',
+      minWidth: 200,
+      maxWidth: 450,
+      isResizable: !disabled,
+      data: 'string',
+      styles: rowTitle(disabled),
+      onRender: item => item.description,
+    },
+    {
+      key: 'column3',
+      name: formatMessage('Created'),
```

Comment:
This column isn't necessary. Please remove

Reply:
Removed. But the design has this column.